scripts/kind-e2e/e2e.sh

```diff
@@ -9,20 +9,20 @@ function kind_clusters() {
             echo Cluster cluster${i} already exists, skipping cluster creation...
         else
             if [[ -n ${version} ]]; then
-                kind create cluster --image=kindest/node:v${version} --name=cluster${i} --wait=5m --config=./kind-e2e/cluster${i}-config.yaml
+                kind create cluster --image=kindest/node:v${version} --name=cluster${i} --wait=5m --config=$(git rev-parse --show-toplevel)/scripts/kind-e2e/cluster${i}-config.yaml
```

Comment:
Just some nitpicking (about making lots of calls...)

Can we capture the directory once at start in a variable and then use the variable?

PRJ_TOPLEVEL=....

also you already have DAPPER_SOURCE that may also work inside dapper... but not outside (if we wanted that..).

Reply:
Wanted to avoid dapper source DAPPER_SOURCE not to depend on it much.

PRJ_TOPLEVEL=....
Great idea!

Reply:
@mangelajo Added PRJ_ROOT and tested the changes.